tinyagentos/routes/user_shares.py

```
@@ -0,0 +1,293 @@
+                source="user_shares",
+                user_id=target_user_id,
+                data={
+                    "share_id": record["id"],
+                    "owner_user_id": user.user_id,
+                    "resource_type": body.resource_type,
+                    "resource_id": body.resource_id,
+                    "permission": body.permission,
+                },
+            )
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)